client/src/pages/Home/Home.tsx

```diff
@@ -63,6 +66,18 @@ function Home(props: any) {
     setCurGeoLocation(position);
   };
 
+  const scanStoreQRCallback = (storeUrl: string) => {
+    // Try to extract store-id and merchant-id fields from storeUrl
+    const storeId = parseUrlParam(storeUrl, "id"); //TODO(#167) Replace with constants for "id"/"mid"
+    const merchantId = parseUrlParam(storeUrl, "mid");
+    // If valid extraction, push us to new site
+    if (storeId && merchantId) {
+      history.push(SCANSTORE_PAGE + "?id=" + storeId + "&mid=" + merchantId);
```

Comment:
nit: Instead of simply appending all the characters, use some URL/URI library to append the params

Reply:
+1

Reply:
yup, definitely agree on this point, might have errors if we happen to change the formatting or arguments passed. This task is tracked with #167, will resolve soon in another PR